poshc2/client/command_handlers/ImplantHandler.py

```diff
@@ -206,7 +207,7 @@ def implant_handler_command_loop(user, printhelp="", autohide=None):
                 do_createdaisypayload(user, command)
                 continue
             if command.startswith("createproxypayload"):
-                do_createproxypayload(user, command)
+                do_createnewpayload(user, command)
```

Comment:
Does this/How will this affect the invoke-XXXproxypayload commands etc? i.e., invoke-wmiproxypayload